dll/cpl/appwiz/lang/it-IT.rc

```diff
@@ -43,14 +43,14 @@ END
 
 IDD_CONFIG_STARTMENU DIALOGEX 0, 0, 212, 200
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
-CAPTION "Remove Shortcuts/Folders"
+CAPTION "Rimuovere Scorciatoie/Cartelle"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "&To remove an item from the Start menu, select the item and click Remove.", -1, 10, 10, 192, 16
+    LTEXT "&Per rimuovere un elemento del menu di avvio, seleziona un elemento e clicca Rimuovi.", -1, 10, 10, 192, 16
```

Comment:
Better replace "seleziona un elemto e ..." with "selezionalo e ..." to avoid repeted words